Added quick start section. init target in Makefile. Updated READMEs (#87)
fredrikhgrelland/vagrant-hashistack-template: README.md

   2. [Testing the module](#testing-the-module)
6. [If This Is in Your Own Repository](#if-this-is-in-your-own-repository)

## Quick start
> :warning: If you are new to the template we strongly recommend you to read the [Description - What & Why](#description---what--why), then perform the steps in [Install Prerequisites](#install-prerequisites). After that you can move on to our getting started guides in the [getting started](#getting-started) section.

You can quickly get started by following the bellow steps:
1. Create your own repository by pressing [Use this template](https://github.com/fredrikhgrelland/vagrant-hashistack-template/generate) button.
2. Clone the repository you just created to your machine.
3. Now, run `make template_init` to clean your repository for unnessesary files and folders.

> 4. You are now all set to start developing you module!

Reviewer: ```suggestion
4. You are now all set to start developing your module!
```

   2. [Testing the module](#testing-the-module)
6. [If This Is in Your Own Repository](#if-this-is-in-your-own-repository)

## Quick start
> :warning: If you are new to the template we strongly recommend you to read the [Description - What & Why](#description---what--why), then perform the steps in [Install Prerequisites](#install-prerequisites). After that you can move on to our getting started guides in the [getting started](#getting-started) section.

> You can quickly get started by following the bellow steps:
> 1. Create your own repository by pressing [Use this template](https://github.com/fredrikhgrelland/vagrant-hashistack-template/generate) button.
> 2. Clone the repository you just created to your machine.
> 3. Now, run `make template_init` to clean your repository for unnessesary files and folders.
> 4. You are now all set to start developing your module!

Reviewer 2: ```suggestion
You can quickly get started by following the below steps:
1. Create your own repository by pressing [Use this template](https://github.com/fredrikhgrelland/vagrant-hashistack-template/generate) button.
2. Clone the repository you just created to your machine.
3. Now, run `make template_init` to clean your repository for unnecessary files and folders.
4. You are now all set to start developing your module!
```

- `make template_example` runs the example in [template_example/](template_example).

> - `make template_init` will clean out the template for you to get started with the module developement.

Reviewer: ```suggestion
- `make template_init` will clean out the template for you to get started with the module development.
```
